release notes/v0.29.0.md

```diff
@@ -0,0 +1,133 @@
+k6 v0.29.0 is here! :tada: It's a feature-packed release with tons of much-requested changes and additions, a lot of them implemented by awesome external contributors! :heart:
+
+As promised in the previous release notes, we're trying to stick to a roughly 8-week release cycle, so you can expect the next k6 version at the start of January 2021, barring any bugfix releases before that.
+
+## New features
+
+### Initial support for gRPC ([#1623](https://github.com/loadimpact/k6/pull/1623))
+
+k6 now supports unary gRPC calls via the new `k6/net/grpc` built-in module. Streaming RPCs are not yet supported and the **JS API is in beta**, so there might be slight breaking changes to the API in future k6 versions, but it's a good start on the road to fully supporting this much-requested protocol!
+
+This is a simple example of how the new module can be used with [grpcb.in](https://grpcb.in/):
+```js
+import grpc from "k6/net/grpc";
+
+let client = new grpc.Client();
+// Download addsvc.proto for https://grpcb.in/, located at:
+// https://raw.githubusercontent.com/moul/pb/master/addsvc/addsvc.proto
+// and put it in the same folder as this script.
+client.load(null, "addsvc.proto");
+
+export default () => {
+    client.connect("grpcb.in:9001", { timeout: "5s" });
+
+    let response = client.invoke("addsvc.Add/Sum", {
+        a: 1,
+        b: 2
+    });
+    console.log(response.message.v); // should print 3
+
+    client.close();
+}
+```
+
+You can find more information and examples how to use k6's new gRPC testing capabilities in [our documentation](https://k6.io/docs/javascript-api/k6-net-grpc).
+
+Huge thanks to @rogchap for adding this feature!
+
+### New options for configuring DNS resolution ([#1612](https://github.com/loadimpact/k6/pull/1612))
+
+You can now control some aspects of how k6 performs DNS resolution! Previously, k6 would have cached DNS responses indefinitely ([#726](https://github.com/loadimpact/k6/issues/726)) and always picked the first resolved IP ([#738](https://github.com/loadimpact/k6/issues/738)) for all connections. This caused issues, especially when load testing services that relied on DNS for load-balancing or auto-scaling.
+
+For technical reasons explored in ([#726](https://github.com/loadimpact/k6/issues/726)), k6 v0.29.0 still doesn't respect the actual TTL value of resolved IPs, that will be fixed in a future k6 version. For now, it simply allows users to specify a global static DNS TTL value and resolution strategy manually. It also has better defaults! Now, by default, the global DNS TTL value is 5 minutes and, if the DNS resolution returned multiple IPs, k6 will pick a random (preferably IPv4) one for each connection.
+
+You can also configure this behavior with the new `--dns` CLI flag, the `K6_DNS` environment variable, or the `dns` script/JSON option. Only 3 DNS resolution options are exposed in this k6 version, `ttl`, `select`, and `policy`.
```

Comment:
I don't really think "Only 3" is necessary

Reply:
Resolved by https://github.com/loadimpact/k6/commit/879d2bf6a58ec4a1ba1d9d1b71b0e0019545bc0d